README.md

```diff
@@ -1,7 +1,17 @@
+# NOTE: Not vulnerable to Log4J 2 "Log4shell"
+
+The Docker images *were* vulnerable to one of a pair of vulnerabilities in Log4J 2 but they are not vulnerable anymore -- you may need to re-pull the image you are using.  For images prior to 8.11.1, Solr is using a popular technique to do this -- setting `log4j2.formatMsgNoLookups`.  The Solr maintainers have deemed this adequate based specifically on how Solr uses logging.  It won't be adequate for all projects that use Log4J.  Scanning software might alert you to the presence of an older Log4J JAR file, however it can't know if your software (Solr) uses the artifacts in a vulnerable way.
```

Comment:
> The Docker images *were* vulnerable to one of a pair of vulnerabilities in Log4J 2 but they are not vulnerable anymore

A lot of the images on hub are still vulnerable. If someone uses e.g. `solr:7.6` they are still vulnerable. They need to upgrade to the latest minor release, e.g. 5.5, 6.6, 7.7, or the latest bugfix release on 8.x, e.g. 8.10.1, but not e.g. [8.10.0](https://hub.docker.com/layers/solr/library/solr/8.10.0/images/sha256-46fef455acab41b7d555638cfa1e1314cb7eb7fd5667302018e5c0e80743ef86?context=explore).

Btw - I see that docker hub has started scanning images for log4shell, perhaps they look for `log4j-core-2.x.jar`. Don't know if they will ban pulls for vulnerable images or what.

Reply:
Maybe you are forgetting/unaware, but there is a mitigation here (in docker-solr) for this: https://github.com/docker-solr/docker-solr/commit/d9aceb632fcad5d5e7ab42b94dd25a008f5b9112 applies to all our tags (thus 8.10 etc.).

Reply:
I'm concerned I had to explain that.  Was the explanation in the text on the page not sufficient?  Maybe I could incorporate a link to the evidence (my commit)?

Reply:
I'm concerned I had to explain that.  Was the explanation in the text on the page not sufficient?  Maybe I could incorporate a link to the evidence (my commit)?

Reply:
> Maybe you are forgetting/unaware, but there is a mitigation here (in docker-solr) for this

Nope, this commit does NOT patch v 7.0, 7.1, 7.2, 7.3, 7.4, 7.5, 7.6. As I say, only the latest minor version of 5.x, 6.x and 7.x are patched, so it is dangerous to write "but they are not vulnerable anymore" as a blanket statement.

Reply:
Ooooh, ok I see your point!  Sorry.  We should communicate the range of Docker images that are vulnerable (which does _not_ include 7.0, 7.1, 7.2, 7.3, that you listed).  It's 7.4 to < 7.7.3.  The recommendation for such users is to upgrade to 7.7.3 (specifying Docker tag 7 or 7.7 is fine) or set that system property.

I will update the README now.

Reply:
#402